docs/core/event_handler/api_gateway.md

```diff
@@ -959,58 +962,23 @@ This will enable full tracebacks errors in the response, print request and respo
-| `le`                  | `float`       | Less than or equal. If set, value must be less than or equal to this. Only applicable to numbers                                                                           |
-| `min_length`          | `int`         | Minimum length for strings                                                                                                                                                 |
-| `max_length`          | `int`         | Maximum length for strings                                                                                                                                                 |
-| `pattern`             | `string`      | A regular expression that the string must match.                                                                                                                           |
-| `strict`              | `bool`        | If `True`, strict validation is applied to the field. See [Strict Mode](https://docs.pydantic.dev/latest/concepts/strict_mode/){target"_blank" rel="nofollow"} for details |
-| `multiple_of`         | `float`       | Value must be a multiple of this. Only applicable to numbers                                                                                                               |
-| `allow_inf_nan`       | `bool`        | Allow `inf`, `-inf`, `nan`. Only applicable to numbers                                                                                                                     |
-| `max_digits`          | `int`         | Maximum number of allow digits for strings                                                                                                                                 |
-| `decimal_places`      | `int`         | Maximum number of decimal places allowed for numbers                                                                                                                       |
-| `examples`            | `List\[Any\]` | List of examples of the field                                                                                                                                              |
-| `deprecated`          | `bool`        | Marks the field as deprecated                                                                                                                                              |
-| `include_in_schema`   | `bool`        | If `False` the field will not be part of the exported OpenAPI schema                                                                                                       |
-| `json_schema_extra`   | `JsonDict`    | Any additional JSON schema data for the schema property                                                                                                                    |
+--8<-- "docs/core/event_handler/\_openapi_customization_parameters.md"
```

Comment:
```suggestion
--8<-- "docs/core/event_handler/_openapi_customization_parameters.md"
```

Reply:
Oops sorry about that, removed all the \_ from the code